features/profiler.yml

```
@@ -0,0 +1,10 @@
+name: Profiler
+description: The `Profiler` API traces and records JavaScript execution on a page, which you can send to a server for later analysis. Also known as self-profiling.
```

Prompted by https://github.com/web-platform-dx/web-features/pull/1458/files#r1726427653

```suggestion
description: The `Profiler` API traces and records script execution, which you can send to a server for later analysis. Also known as self-profiling.
```

---

Now that I think of it, there's an extension to the API that's happening where paint/layout/style/gc/... "markers" are now also part of the profile. See https://github.com/WICG/js-self-profiling/blob/main/markers.md
I think this is sufficiently different to be covered by a separate feature later. But I'm tempted to rephrase to:

```suggestion
description: The `Profiler` API records data about the execution of a page, which you can send to a server for later analysis. Also known as self-profiling.
```

---

👍 That works for me. And if we decide they're separate feature, we can always re-work the name and description.